compute_endpoint/tests/integration/endpoint/endpoint/test_endpoint_manager.py

```diff
@@ -436,44 +436,60 @@ def test_with_funcx_config(self, mocker):
             **mock_optionals,
         )
 
-    @pytest.mark.parametrize("dir_exists", (True, False))
+    @pytest.mark.parametrize("dir_exists", (False,))
```

Comment:
We can drop this parameter since it only has a single value and doesn't appear to be utilized.

Reply:
Good catch.  Was previously True/False and accidentally removed the True option while debugging.  Added it back as more coverage is good.

Reply:
Sounds good, but the parameter is also not utilized in the function.

Reply:
Now that's funny.  Might be I refactored this into the other tests earlier and forgot to remove this one, which I just did.